r/R/embedding.R

```diff
@@ -83,3 +61,67 @@ runEmbedding <- function(req, data) {
   res <- purrr::map2(df_embedding[[1]], df_embedding[[2]], map2_fun)
   return(res)
 }
+
+
+# getEmbedding
+# Return embedding calculated for the seurat object.
+# data is the seurat object.
+# config is the embedding work request config.
+# type is the embedding type.
```

Comment:
Given that the name of the variable has been changed [here](https://github.com/hms-dbmi-cellenics/worker/pull/287/files#:~:text=method%20%3C%2D%20req%24body%24type) I think it should be changed also here
```suggestion
# method is the embedding type.
```